packages/frontend/src/components/ExportSelector/index.tsx

```diff
@@ -33,8 +36,13 @@ const ExportSelector: FC<
         );
     } else if (hasGoogleDrive && getGsheetLink) {
         return (
-            <Stack>
-                <Button variant="subtle" onClick={() => setExportType('csv')}>
+            <Stack spacing="xs">
+                <Button
+                    size="xs"
+                    variant="default"
+                    disabled={!!isGoogleSheetsExporting}
```

Comment:
button should be disabled if `react-query` can detect there's a `google-sheets` query still running